starkcore/utils/checks/checks.go

```diff
@@ -0,0 +1,51 @@
+package checks
+
+import (
+	"fmt"
+	"github.com/starkbank/ecdsa-go/ellipticcurve/privatekey"
+	"github.com/starkinfra/core-go/starkcore/environment"
+	"github.com/starkinfra/core-go/starkcore/utils/hosts"
+	"reflect"
+	"strings"
+)
+
+func CheckEnvironment(env string) string {
+	var acceptedEnvironments []string
+	v := reflect.ValueOf(environment.Environments)
+	for i := 0; i < v.NumField(); i++ {
+		acceptedEnvironments = append(acceptedEnvironments, v.Field(i).String())
+		if env == v.Field(i).Interface() {
+			return env
+		}
+	}
```

Comment:
nao rolava fazer um` for _, e := range environment.Environments` aqui pra simplificar um pouco não?

Reply:
Não, porque environment.Environments não é iterável. A única forma de fazer a iteração de uma struct é desta forma.